src/shared/ui/SearchForm/SreachForm.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { getByTestId, render, screen } from "@testing-library/react";
+import userEvent from '@testing-library/user-event'
+import { SearchForm } from './SearchForm';
+
+describe("SearchForm Component Functionality", () => {
+    test("Should render initial value", () => {
+        const initialValue = "Hello World!";
+        const OnSearchClick = jest.fn();
```

Comment:
in JS we use camel case 
```suggestion
        const onSearchClick = jest.fn();
```

Reply:
changed